.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -245,10 +245,6 @@ jobs:
     - name: Create bundle output directory
       run: mkdir -p definitions/${{ github.event.pull_request.number }}
 
-    - name: Run Updater (single bundle)
```

Comment:
I think we still need this for 4.4s ?

Reply:
yes, but this is for PRs, and we don't actively work on 4.4 anymore (also Scanner V4 was Tech Preview then, so I do not believe we are on the hook to fix any problems)